Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOne/TrendMicroVisionOne.py

```diff
@@ -167,23 +167,25 @@ def check_datetime_aware(d):
 
 
 class Client(BaseClient):
-    def __init__(self, base_url: str, api_key: str) -> None:
+    def __init__(self, base_url: str, api_key: str, proxy: bool) -> None:
         """
         Inherit the BaseClient class from the demistomock.
         :type base_url: ``str``
         :param base_url: Base server address with suffix, for example: https://example.com/api/v2/.
         :type api_key: ``str``
         :param api_key: api token to access the api data.
-        :type verify: ``bool``
-        :param verify: Whether the request should verify the SSL certificate.
+        :type proxy: ``bool``
+        :param proxy: Whether the request should use the system proxy settings.
         :return: returns None
         :rtype: ``None``
         """
-        super().__init__(base_url=base_url)
         self.base_url = base_url
         self.api_key = api_key
+        self.proxy = proxy
```

Comment:
Why do you need this?
```suggestion
```

Reply:
Nope. its ok I'll fix it.
I just wanted to make sure it was really unnecessary :).

Thanks!